server/lib/nl/config_builder/bar.py

```diff
@@ -118,10 +118,14 @@ def multiple_place_bar_block(column,
 
 
 def get_sort_order(state: PopulateState, cspec: ChartSpec):
-  # Use no default sort_order for special DC, since UN
-  # want the order to be as provided in variable groupings.
-  sort_order = None if is_special_dc(state.uttr.insight_ctx) \
-    else BarTileSpec.DESCENDING
+  sort_order = None
+
+  # No default sort_order for special DC since UN wants the order to be as
+  # provided in variable groupings. Also no default sort order if there is more
+  # than one sv because for cases like age ranges, we also want the order to be
+  # what's provided in the variable groupings.
+  if not is_special_dc(state.uttr.insight_ctx) and cspec.svs == 1:
+    sort_order = BarTileSpec.DESCENDING
```

Comment:
Could we instead disable the sorting for cases where there are numbers in the SV (e.g. age ranges, number of rooms, income ranges etc). This seems like a regression for other cases, such as modes of commute or job families  which are more useful when they are sorted.

Reply:
updated to only cases where there is a range in the SVs